tests/lib/rules/indent.js

```diff
@@ -838,7 +839,12 @@ ruleTester.run("indent", rule, {
                 "b();\n" +
                 "}\n",
             options: [2],
-            errors: expectedErrors([[3, 2, 0, "ExpressionStatement"]])
+            errors: expectedErrors([[3, 2, 0, "ExpressionStatement"]]),
```

Comment:
Can you put `output` next to `code` in the tests? Makes it easier to see issues.